packages/block-library/src/query/edit/query-inspector-controls.js

```diff
@@ -143,8 +144,21 @@ export default function QueryInspectorControls( {
 		onChangeDebounced();
 		return onChangeDebounced.cancel;
 	}, [ querySearch, onChangeDebounced ] );
+
+	const CreateNewPostLink = ( { type } ) => {
+		const newPostUrl = addQueryArgs( 'post-new.php', {
+			post_type: type,
+		} );
+		return (
+			<div className="wp-block-query__create-new-link">
+				<a href={ newPostUrl }> { __( 'Create a new post' ) }</a>{ ' ' }
```

Comment:
Needs better handling of translations here. Should I wrap all the text in a link like other cases in GB (ex `Learn more about manual excerpts`).